Doc/library/shutil.rst

```diff
@@ -230,9 +230,9 @@ Directory and files operations
               dirs_exist_ok=False)
 
    Recursively copy an entire directory tree rooted at *src* to a directory
-   named *dst* and return the destination directory. *dirs_exist_ok* dictates
-   whether to raise an exception in case *dst* or any missing parent directory
-   already exists.
+   named *dst* and return the destination directory.  All intermediate
+   directories needed to contain *dst* will also be created by default (see
+   kwarg *dirs_exist_ok*).
```

Comment:
"kwarg" is a bit too informal. Maybe the parenthetical should just be removed?

Reply:
I'm ok with that, it's removed in c169b1d3d2d67e9be37a95b3843896b0bd3070d9